src/dropdown.test.basics.multiple.ts

```diff
@@ -367,3 +342,56 @@ it('does not include in its `value` disabled options that are selected', async (
 
   expect(host.value).to.deep.equal([]);
 });
+
+it('only selects the first option with a matching value when multiple options have the same value and `value` is set', async () => {
```

Comment:
> When `value` is set programmatically, only the first Dropdown Option with a matching value is now selected instead of every Dropdown Option with a matching value.